nuxeo-runtime/nuxeo-stream/src/main/java/org/nuxeo/lib/stream/log/kafka/KafkaLogTailer.java

```diff
@@ -92,7 +92,12 @@
 
     protected boolean consumerMoved;
 
-    protected KafkaLogTailer(KafkaNamespace ns, String group, Properties consumerProps) {
+    protected KafkaLogTailer(Codec<M> codec, KafkaNamespace ns, String group, Properties consumerProps) {
+        if (codec == null) {
+            this.codec = new SerializableCodec<>();
```

Comment:
Maybe put `new SerializableCodec<>()` in a `static final` constant to not instantiate one each time?

Reply:
This is not possible to use a static because of the generics usage, this object is small and tied to the tailer life it should be ok.